project/src/main/java/com/google/sps/data/PlacesFetcher.java

```diff
@@ -146,7 +136,7 @@ PlaceDetails getPlaceDetails(PlaceDetailsRequest request)
         return request.await();
     }
 
-    private static String createCuisinesQuery() {
-        return String.join("|", CUISINES);
+    private static String createCuisenesQuery(ImmutableList<String> cuisines) {
```

Comment:
Typo somehow returned..

Reply:
Yes because this was ready before the typo fix was merged and then when merging I stayed with this version. I will notice those things next times.